hack/e2e.sh

```diff
@@ -89,18 +99,30 @@ PE_URL=$(oc get route cincinnati-policy-engine -o jsonpath='{.spec.host}')
 export GRAPH_URL="http://${PE_URL}/api/upgrades_info/v1/graph"
 
 # Wait for route to become available
-ATTEMPTS=10
 DELAY=10
-
-while [ $ATTEMPTS -ge 0 ]; do
+for i in $(seq 10); do
```

Comment:
I'd prefer being explicit here:

```suggestion
for i in $(seq 1 10); do
```